procedures/auxtel/observation_procedures/Parabolic_Focus_Sequence.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    Filter_to_focus = 'SDSSg'
</pre><p>Maybe make this SDSSr since g probably won't go back in the instrument</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Thanks! I also changed the variable name to lowercase to reflect python pep8 standard. </p>